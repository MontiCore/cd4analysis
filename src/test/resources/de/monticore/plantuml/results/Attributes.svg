<?xml version="1.0" encoding="us-ascii" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentStyleType="text/css" height="112px" preserveAspectRatio="none" style="width:208px;height:112px;background:#FFFFFF;" version="1.1" viewBox="0 0 208 112" width="208px" zoomAndPan="magnify"><defs/><g><rect fill="none" height="27.6094" id="_legend" style="stroke:#000000;stroke-width:1.0;" width="98" x="64.25" y="12"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="88" x="69.25" y="31.5332">.Attributes CD</text><!--class Person--><g id="elem_Person"><rect codeLine="19" fill="#FFFFFF" height="27.6094" id="Person" style="stroke:#181818;stroke-width:0.5;" width="51" x="32.75" y="59.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="45" x="35.75" y="79.1426">Person</text></g><!--class Alphabet--><g id="elem_Alphabet"><rect codeLine="21" fill="#FFFFFF" height="27.6094" id="Alphabet" style="stroke:#181818;stroke-width:0.5;" width="62" x="119.25" y="59.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="56" x="122.25" y="79.1426">Alphabet</text></g><text fill="#888888" font-family="sans-serif" font-size="10" lengthAdjust="spacing" textLength="200" x="0" y="102.5996">generated with MontiCore using PlantUML</text><!--SRC=[RKz1QWCn33olNw4Nf7r0AAJ9fT10I0azUxsgBUAL5rbBIKl-Ndkz50hrGPP6CyDec_KMySbqBjgS3sIzs16-JTU0fUkUhRjwIJCF-nIJm7iWnReuDqmO1HvD1t2l9Q977eeKblU_kkt-6bJaRhOFQwP00u8ZSH5RZ-EaDnXnx56oAVHy9PviJRoatQaAzRFYYqlyHbzOFDSmcYOGya6BQlFBox0_cBfkBDDkFXNtn6QTTd4AjaSjCxAMd1yfrQy8KApY09-a0Ow9bSeb27Cczd2AblLoVFq1]--></g></svg>
