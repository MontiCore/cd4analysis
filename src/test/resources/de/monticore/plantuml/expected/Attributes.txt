@startuml
<style>
	classDiagram {
		class {
			BackgroundColor White
			RoundCorner 0
	  }
	  legend {
      BackgroundColor White
      RoundCorner 0
    }
</style>
hide circle
hide empty members

  skinparam classAttributeIconSize 0
legend top right
  .Attributes CD
end legend
class Person {
  {field} public String name
  {field} protected int age
  {field} private String adress
  {field} final long id
  {field} readonly String birthplace
  {field} List<Person> children
  {field} derived Person parent1
  {field} derived Person parent2
  {field} java.util.Date birthday
  {field} boolean happy
  {field} private double bankBalance
}

class Alphabet {
  {field} List<char> alphabet
  {field} int length
  {field} List<String> umlaute
}

center footer generated with MontiCore using PlantUML
@enduml
